ruby-challenges.rb

```diff
@@ -4,24 +4,46 @@
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
 num1 = 7
-# Expected output: '7 is odd'
+ #Expected output: '7 is odd'
 num2 = 42
 # Expected output: '42 is even'
 num3 = 221
 # Expected output: '221 is odd'
 
+def even_or_odd (num1, num2, num3)
+    if num1 === 7
+        puts "#{num1} is odd"
+    elsif num2 === 42
+        puts "#{num2} is even"
+    elsif num3 === 221
+        puts "#{num3} is odd" 
+    end
+     
+end
 
+p even_or_odd
+
+#-------------------- terminal returned: syntax error, unexpected end-of-input, expecting `end', unsure whats going on, troubleshooted and made it worse! : `even_or_odd': wrong number of arguments (given 0, expected 3) (ArgumentError)
```

Comment:
This error is stating that you have 3 parameters set up in your method, but passing no arguments when invoking the method.  Methods can make our code really dynamic and so we want to utilize that power by passing only one argument and setting up one param for this method and avoiding hardcoding values.  I would recommend reviewing the [Ruby Methods](https://github.com/learn-academy-2022-delta/Syllabus/blob/main/ruby/methods.md) section in the syllabus.